engine/sandboxedGames.js

```diff
@@ -71,11 +82,25 @@ define([
 			this.timeout = null;
 			this.waiting = true;
 			this.lastStartStep = Date.now();
+			let notForward = (type !== null && steps === null);
```

Comment:
The condition here has changed from `type !== null || steps !== null` to `type !== null && steps === null`. This seems a strange change. I haven't looked over the logic in detail, but was this an old bug which has been fixed? a typo? something else?